streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/impl/KubernetesNativeApplicationClient.scala

```diff
@@ -62,14 +67,17 @@ object KubernetesNativeApplicationClient extends KubernetesNativeClientTrait {
     var clusterDescriptor: KubernetesClusterDescriptor = null
     var clusterClient: ClusterClient[String] = null
 
+    val spec = convertFlinkDeploymentDef(submitRequest, flinkConfig)
+
     try {
       val (descriptor, clusterSpecification) = getK8sClusterDescriptorAndSpecification(flinkConfig)
       clusterDescriptor = descriptor
-      val applicationConfig = ApplicationConfiguration.fromConfiguration(flinkConfig)
-      clusterClient = clusterDescriptor
-        .deployApplicationCluster(clusterSpecification, applicationConfig)
-        .getClusterClient
 
+      val applicationConfig = ApplicationConfiguration.fromConfiguration(flinkConfig)
+//      clusterClient = clusterDescriptor
+//        .deployApplicationCluster(clusterSpecification, applicationConfig)
+//        .getClusterClient
+      FlinkK8sOperator.deployApplicationJob(submitRequest.id, spec)
```

Comment:
FlinkK8sOperator.deployApplicationJob is an IO strucuture, need to invoke .runIO to run it actually.

Reply:
At present, it seems that the client module will throw an exception when calling the FlinkK8sOperator.deployApplication Job(submit Request.id, spec) API.